src/tokens/evo-core.css

```diff
@@ -93,6 +93,7 @@
     --font-size-18: 1.125rem;
     --font-size-20: 1.25rem;
     --font-size-24: 1.5rem;
+    --font-size-28: 1.75rem;
```

Comment:
👀  Not sure if I should add this here, but our Large Text typography style is size 20 / line height 28.